pkgs/by-name/tf/tftui/package.nix

```diff
@@ -18,6 +18,11 @@ python3.pkgs.buildPythonApplication rec {
     hash = "sha256-xOlPuPVwfVT7jfBJPqZ5FbOs80HE0k2ZqcA+Jcxh9p4=";
   };
 
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace "posthog = \"^3.1.0\"" "posthog = \"^4.0.0\""
+  '';
+
```

Comment:
Have you tried adding it to `pythonRelaxDeps` below?

Reply:
That was the first thing I tried, without any luck.

Reply:
Also, `python3.pkgs` -> `python3Packages` done.

Reply:
For me adding `"posthog"` to the pythonRelaxDeps list works just fine

Reply:
I swear I tried it, but let me try it again.

Reply:
Huh, worked this time. Done.